src/dotnet/APIView/apiview_token_gist.json

```diff
@@ -0,0 +1,356 @@
+            "NavigateToId": null,
+            "Value": "ClassLibrary1",
+            "Kind": 0
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": " ",
+            "Kind": 2
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": "{",
+            "Kind": 3
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": null,
+            "Kind": 1
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": "    ",
+            "Kind": 2
+        },
+        {
+            "DefinitionId": "ClassLibrary1.Class1",
+            "NavigateToId": null,
+            "Value": null,
+            "Kind": 5
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": "public",
+            "Kind": 4
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": " ",
+            "Kind": 2
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": "class",
+            "Kind": 4
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": " ",
+            "Kind": 2
+        },
+        {
+            "DefinitionId": "ClassLibrary1.Class1",
+            "NavigateToId": null,
+            "Value": "Class1",
+            "Kind": 6,
+            "CrossLanguageDefId": "Namespace1.Class1"
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": " ",
+            "Kind": 2
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": "{",
+            "Kind": 3
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": null,
+            "Kind": 1
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": "        ",
+            "Kind": 2
+        },
+        {
+            "DefinitionId": "ClassLibrary1.Class1.Class1(System.String, System.Int32)",
+            "NavigateToId": null,
+            "Value": null,
+            "Kind": 5
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": "public",
+            "Kind": 4
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": " ",
+            "Kind": 2
+        },
+        {
+            "DefinitionId": "ClassLibrary1.Class1.Class1(System.String, System.Int32)",
+            "NavigateToId": null,
+            "Value": "Class1",
+            "Kind": 6,
+            "CrossLanguageDefId": "Namespace1.Class1.Class1"
```

Comment:
@lmazuel and this is for method/constructor level. This ID format can be used for public properties/variables  within a class also.

Reply:
If there is an overload of this method, how would the CrossLangaugeDefId be a) deterministic b) unique?

Reply:
Then we need further classifier with a number suffix if every language will have taht overloaded API.